README.md

```diff
@@ -16,53 +16,58 @@ This package implements various 3D rotation parameterizations and defines conver
     # create a point
     X = Vec(1.0, 2.0, 3.0)
 
-    # convert to Euler Angles (using the default ordering scheme)
+    # convert to a Quaternion and rotate the point
+    q = Quaternion(R)
+    Xo = rotate(ea, X)
+
+    # convert to a Stereographic quaternion projection (recommended for optimization problems) and rotate
+    spq = SpQuat(R)
+    Xo = rotate(spq, X)
+
+    # convert to a Rodrigues Vector and rotate
+    rv = RodriguesVec(R)
+    Xo = rotate(rv, X)
+
+    # convert to Euler Angles (using the default ordering scheme) and rotate
     ea = EulerAngles(R)
     Xo = rotate(ea, X)
 
-    # convert to Euler Angles specifying the angle order
+    # convert to Euler Angles (specifying an angle order) and rotate
     ea = EulerAngles{Rotations.EulerXYZ}(R)
     Xo = rotate(ea, X)
 
-    # convert to proper Euler Angles (using the default ordering scheme)
+    # convert to proper Euler Angles (using the default ordering scheme) and rotate
     ea = ProperEulerAngles(R)
     Xo = rotate(ea, X)
 
-    # convert to proper Euler Angles specifying the angle order
+    # convert to proper Euler Angles (specifying an angle order) and rotate
     ea = ProperEulerAngles{Rotations.EulerXYX}(R)
     Xo = rotate(ea, X)
 
-    # a Quaternion
-    q = Quaternion(R)
-    Xo = rotate(ea, X)
-
-    # Stereo graphic projection of a quaternion (recommended for optimization problems)
-    spq = SpQuat(R)
-    Xo = rotate(spq, X)
 
 ```
 
 ### Rotation Parameterizations
 
-1. **Rotation Matrix** `RotMatrix{T <: AbstractFloat}`
+1. **Rotation Matrix** `RotMatrix{T <: Real}`
```

Comment:
Should this be `Number`?

e.g. `DualNumber <: Number`. If people want to put in strange things, that's their business...